governance-exercises/JanSky Polling Exercise 4 - Deploy Emergency Fix A in imminent executive -  August 9, 2023.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Deploy Emergency Fix A in imminent executive -  August 9, 2023
+summary: Signal your support or opposition to deploying Fix A (an immediate but costly remedy) in reaction to a verified bug.
+discussion_link: $discussion_link
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-08-09T18:00:00
+end_date: 2023-08-10T06:00:00
+---
+# Poll: Deploy Emergency Fix A in imminent executive -  August 9, 2023
```

Comment:
Given this will also be consumed by the wider crypto space. It is worth highlighting that no customer funds are at risk, etc. Basically repeating the key info in Imogen's post.